server/src/db.rs

```diff
@@ -558,6 +559,41 @@ impl Db {
         tracker
     }
 
+    /// Spawns a task to perform
+    /// [`load_chunk_to_object_store`](Self::load_chunk_to_object_store)
+    pub fn load_chunk_to_object_store_in_background(
```

Comment:
I wonder if this might be more consistently named  `write_chunk_to_object_store_in_background` (consistent with `Job::WriteChunk`)?

```suggestion
    pub fn write_chunk_to_object_store_in_background(
```

(though I realize you are following the pattern of `load_chunk_to_object_store`)

Reply:
Let me do that last, AFTER resolving all other issues and test questions to avoid rebasing back and forth.